markdown/org/docs/patterns/lunetius/cutting/en.md

```diff
@@ -1,3 +1,21 @@
-Lunetius consists only of one part, the *lacerna* itself:
+---
+title: Cutting Lunetius
```

Comment:
Please use `Cutting` as title because that's what we use everywhere.
Although I prefer it this way, I think for now we'll stick with this for consistency

Reply:
Done!